agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -620,21 +620,6 @@ private static void applyDatabaseClientSpan(
       } else {
         type = "SQL";
       }
-      // keeping existing behavior that was release in 3.0.0 for now
-      // not going with new jdbc instrumentation span name of
-      // "<db.operation> <db.name>.<db.sql.table>" for now just in case this behavior is reversed
-      // due to spec:
-      // "It is not recommended to attempt any client-side parsing of `db.statement` just to get
-      // these properties, they should only be used if the library being instrumented already
-      // provides them."
-      // also need to discuss with other AI language exporters
-      //
-      // if we go to shorter span name now, and it gets reverted, no way for customers to get the
-      // shorter name back
-      // whereas if we go to shorter span name in the future, and they still prefer more
-      // cardinality, they can get that back using telemetry processor to copy db.statement into
-      // span name
-      telemetry.setName(dbStatement);
```

Comment:
no need to track dbStatement anymore? 

"select * from abc" makes more sense than "select mysql.abc"? (from the tests)

Reply:
any link to the spec change?

Reply:
> any link to the spec change?

ya, https://github.com/microsoft/common-schema/pull/286

Reply:
> If [Span.dbStatement](../PartB/traces/Span.md#dbstatement) is not available then use `dbOperation`.
should we keep dbStatement when it's available?

> "select * from abc" makes more sense than "select mysql.abc"? (from the tests)
thoughts?

Reply:
we still put dbStatement into the data field

Reply:
> telemetry.setData(dbStatement);

I see. can we update the test to verify the dbStatement instead of getName()? since dbStatement is no longer part of name.

Reply:
we verify both